modal.css

```diff
@@ -0,0 +1,312 @@
+    margin-top: -25px;
+    left: 24px;
+  }
+
+  .side-info p {
+    width: 311px;
+    height: 220px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+    color: #344563;
+    margin-top: 0;
+  }
+
+  .popup-lang a {
+    text-decoration: none;
+    border: 1px solid #979493;
+    padding: 10px 12px;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-weight: bold;
+    font-size: 13px;
+    color: #3c3a39;
+    line-height: 20px;
+    width: 39px;
+    height: 20px;
+  }
+
+  .out-links a {
+    width: 220px;
+    margin-top: 15px;
+    left: 708px;
+    padding-bottom: 10px;
+    margin-right: 12px;
+    background-color: #f55800;
+    text-decoration: none;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 3%;
+    text-align: center;
+  }
+
+  .popup-lang a:hover {
+    background-color: #f55800;
+    color: white;
+  }
+
+  .out-links a:hover {
+    background-color: white;
+    color: #f55800;
+  }
+
+  .out-links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    right: 50%;
+  }
+
+  .out-links img {
+    padding-left: 10px;
+    padding-top: 20px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #modal-box {
+    background-color: rgb(255, 255, 255);
+    width: 70%;
+    height: auto;
+    max-height: 75%;
+    max-width: 1156px;
+    top: 112px;
+    left: 142px;
+    box-shadow: 0 0 rgba(241, 90, 34, 0.8);
+    transform: scale(1);
+    padding: 30px;
+    z-index: 99999;
+    transition: 200ms ease-in-out;
+  }
+
+  #modal-box.active {
+    transform: scale(1);
+  }
+
+  #modal-wrapper {
+    display: none;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.7);
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    backdrop-filter: blur(10px);
+  }
+
+  #modal-wrapper.active {
+    display: flex;
+  }
+
+  #modal-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  #modal-box #close-modal {
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: none;
+    font-size: 2.25rem;
+    font-weight: bold;
+    color: #333;
+  }
+
+  #modal-header h2 {
+    width: 1056px;
+    width: 70%;
+    height: 70%;
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    margin-top: 0;
+  }
+
+  .popup-lang {
+    display: flex;
+    gap: 10px;
+    margin-left: -40px;
+    list-style-type: none;
+    margin-top: 0;
+    margin-bottom: 50px;
+    width: 70%;
+  }
+
+  .popup-details {
+    display: flex;
+    gap: 12px;
+  }
+
+  .feature-image {
+    width: 100%;
+    height: 400px;
+    top: 156px;
+    left: 24px;
+    bottom: 24px;
+  }
```

Comment:
- [ ] Great job incorporating the changes suggested by the previous reviewer. However, while implementing those changes, it appears that a new bug has been introduced into your code. The bug causes your `model` to become less responsive, resulting in content overflowing and potentially affecting the user experience. To fix this issue kindly consider the below suggestions:
 - [ ] Kindly consider setting the image's `height` property to `100%` instead of `400px`. By using `height: 100%`, the image will become responsive and adjust its height dynamically based on the screen size. This ensures that the image scales proportionally and maintains its aspect ratio, providing a better user experience across different devices and screen resolutions. See below for more details:
 <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="610" alt="Screenshot 2023-07-01 at 10 03 34" src="https://github.com/anyars-encarta/anyars-encarta.github.io/assets/107192076/21c15396-eaa0-4566-86ef-95af37e40f14">
</td>
<td>
<img width="610" alt="Screenshot 2023-07-01 at 10 03 34" src="https://github.com/anyars-encarta/anyars-encarta.github.io/assets/107192076/f6b432d6-4ac6-4ee8-aa62-5f88baca2d09">
</td>
</tr>
</table>